sdk/storage/azure-storage-blob-changefeed/src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1 @@
+mock-maker-inline
```

Comment:
what does it do?

Reply:
I looked up how to make Mockito work with Groovy and this is what came of it.